src/data/clients.ts

```diff
@@ -863,5 +863,28 @@ export const Clients: Array<Client> = [
         url: 'https://fintunes.app'
       }
     ]
+  },
+  {
+    id: 'ampfin',
+    name: 'AmpFin',
+    description: 'iOS 17 native Jellyfin music player',
+    clientType: ClientType.ThirdParty,
+    deviceTypes: [DeviceType.Mobile],
+    licenseType: LicenseType.Proprietary,
```

Comment:
Is this correct?

Reply:
Yes, it Is. The app is licensed under the Mozilla Public License 2 but the Common Clause also applies. So the app is only source available, and I think calling it open source would be misleading. Proprietary is a more accurate description.